apps/backoffice-v2/src/domains/workflows/query-keys.ts

```diff
@@ -30,6 +30,10 @@ export const workflowsQueryKeys = createQueryKeys('workflows', {
       queryFn: () => fetchWorkflows(data),
     };
   },
+  workflowDefinition: ({ workflowDefinitionId }: { workflowDefinitionId: string }) => ({
```

Comment:
If separating to a different keys object is too much let's at least add a `ById` suffix.

Reply:
It has to be in its own query key file indeed. But i dont think its worth it to generate lots of boilerplate just for single endpoint in current case. Rename should work.